### DISCUSSION TITLE
v1.111.0 - build 1

### DISCUSSION THREAD

**Discussion by 4v3ngR** [4v3ngR/immich-native-macos#8] [Announcements]
Alpha software

<hr /><em>This discussion was created from the release <a href='https://github.com/4v3ngR/immich-native-macos/releases/tag/'>v1.111.0 - build 1</a>.</em>

**Comment by hornofabraxas** [comment]
On m1 mac mini, whether I build the .pkg from build.sh or run the release pkg, I get Installation failed: 

`/tmp/build.sh: line 40: npm: command not found`

I have homebrew and all the dependencies (including node) install just fine. Unsure how to proceed

**Comment by hornofabraxas** [comment]
Had to modify the build.sh script to add the NPM path, then it ran the script and supposedly installed successfully
However after install I could not access any services on port 3001
Running uninstall script told me there was no "immich" database
Looks like this does not work on m1, or has dependencies not specified here.
